pod/perl.pod

```diff
@@ -391,10 +391,13 @@ see L<perlvar> for more information.
 
 Using the C<use strict> pragma ensures that all variables are properly
 declared and prevents other misuses of legacy Perl features.
+These are enabled by default within the scope of
+C<L<use v5.12|perlfunc/use VERSION>> (or higher).
 
-The C<use warnings> pragma produces some lovely diagnostics. One can
-also use the B<-w> flag, but its use is normally discouraged, because
-it gets applied to all executed Perl code, including that not under
+The C<use warnings> pragma produces some lovely diagnostics.
```

Comment:
_lovely_ I do not believe is an appropriate descript since it is meaningless in this context. Can we get a more precise adjective about what is meant? Works like, _actionable_, _informative_, _helpful_, etc. With the audience in mind (a developer), these are the kinds of things I want to be able to know.

Reply:
Text was already present, so not really in scope for this PR.